docs/src/tutorial.jl

```diff
@@ -680,12 +680,11 @@ adf
 
 # In realistic modelling situations it is often the case the the ABM is composed
 # of different types of agents. Agents.jl supports two approaches for multi-agent ABMs.
-# The first uses the `Union` type, and the second
-# uses the [`@multiagent`](@ref) command. `@multiagent` is recommended
-# as default, because in many cases it will have performance advantages over the `Union` approach
-# without having tangible disadvantages. However, you should read through
-# the [comparison of the two approaches](@ref multi_vs_union) to
-# be better informed on which one to choose.
+# The first uses the `Union` type (this subsection), and the second uses the [`@sumtype`](@ref)
+# command (next subsection) available in [DynamicSumTypes.jl](https://github.com/JuliaDynamics/DynamicSumTypes.jl). 
+# This approach is recommended as default, because in many cases it will have performance advantages 
+# over the `Union` approach without having tangible disadvantages. However, we strongly recommend you 
```

Comment:
```suggestion
# over the `Union` approach without having tangible disadvantages. However, we recommend you 
```
we shouldn't recommend this strongly. If for the typical user it doesn't matter to use the recommended way, there is no "strong" incentive to attempt to learn more about this advanced topic